packages/ddp-client/livedata_connection.js

```diff
@@ -703,6 +737,7 @@ _.extend(Connection.prototype, {
    * @param {Object} [options]
    * @param {Boolean} options.wait (Client only) If true, don't send this method until all previous method calls have completed, and don't send any subsequent method calls until this one is completed.
    * @param {Function} options.onResultReceived (Client only) This callback is invoked with the error or result of the method (just like `asyncCallback`) as soon as the error or result is available. The local cache may not yet reflect the writes performed by the method.
+   * @param (Boolean) options.noRetry (Client only) if true, don't send this method again on reload, simply call the callback with the error
```

Comment:
The documentation is generated from here, see how to update it in this hackpad: https://meteor.hackpad.com/Automatically-Generating-API-Docs-using-JSDoc-EpPmd2iuFEH